tests/e2e/tests/pilot/testdata/networking/v1alpha3/rule-default-route-append-headers.yaml

```diff
@@ -11,5 +11,11 @@ spec:
           host: c
           subset: v1
         weight: 100
+        append_headers:
+          istio-destination-custom-header: user-defined-value
+        append_response_headers:
+          istio-destination-custom-header-response: user-defined-value
+        remove_response_headers:
+        - istio-destination-custom-header-remove
```

Comment:
where is the check? You need to check the value in the code.

Reply:
There wasn't test for append headers, so I just added new test for route append headers and the my new code :)